src/routes/genre.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const genreController = require('../controllers/genre');
+
+const router = express.Router();
+
+router.post('', genreController.addGenre);
+router.get('', genreController.getAllGenres);
```

Comment:
Although there is nothing wrong with this, it is considered best practice to put a / between the quotations in the first parameter if there is no specific route after the base route
```
router.post('/', genreController.addGenre);
router.get('/', genreController.getAllGenres);
```